src/App.js

```diff
@@ -1,29 +1,13 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable */
```

Comment:
- [Optional] It is not a good practice to disable Linters in your codebase. Always endeavor to fix bugs flagged by Linters to keep your code clean.